src/test/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.figure;
+
+public abstract class Figure {
+
+    public String color;
+    public String name;
+
+    public abstract void getDraw();
+
+    public abstract void setRandomParameter();
+
+    public abstract void setDefaultParameter();
```

Comment:
remove these methods, they don't need you, you should pass parameters to the constructors of Figure heirs